tests/modules/images/webp-uploads/helper-tests.php

```diff
@@ -270,7 +270,33 @@ function () {
 		$this->assertArrayHasKey( 'filesize', $result );
 		$this->assertArrayHasKey( 'file', $result );
 		$this->assertStringEndsWith( 'image.webp', $result['file'] );
-		$this->assertFileExists( '/tmp/image.webp' );
+	}
+
+	/**
+	 * Tests the webp_uploads_pre_generate_additional_image_source filter returning filesize property.
+	 *
+	 * @test
+	 */
+	public function it_should_return_an_error_when_filter_webp_uploads_pre_generate_additional_image_source_returns_filesize() {
```

Comment:
This name is not quite right as far as I can tell? Where in this test is there any error to be returned?